nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -19,9 +19,7 @@ The above instructions don't deal with the appendix of available `configuration.
 
 ## Testing redirects {#sec-contributing-redirects}
 
-Once you have a successful build, you can open the relevant HTML (path mentioned above) in a browser along with the anchor, and observe the redirection.
-
-Note that if you already loaded the page and *then* input the anchor, you will need to perform a reload. This is because browsers do not re-run client JS code when only the anchor has changed.
+Check further context and instructions in the [Nixpkgs manual docs README](../../../doc/README.md#redirects-check).
```

Comment:
It's really just one or two sentences of overlap which IMO we should write out here, and then do a full example for nixos modules, and adding aliases and what not.